libs/core/src/components.d.ts

```diff
@@ -1232,6 +1232,7 @@ declare global {
     };
     interface HTMLPdsChipElementEventMap {
         "pdsTagCloseClick": any;
+        "pdsClick": any;
```

Comment:
I’m not entirely sure since we don’t have a dropdown component implemented yet, but given that this is meant to act as a dropdown trigger, should the dropdown itself handle the click event? Just wondering if there’s a risk of multiple conflicting events being fired.

Just a thought.

Reply:
Fair point. I suppose it really depends on how we're building the Dropdown and whether we'd like to still consider the `panel` as a separate component, e.g. something we can use in multiple components like Dropdown, Popover, Modal, etc.

Reply:
I'm going to bring this up as a topic of discussion once we have a meeting surrounding pattern work soon.